fabric/src/main/java/dev/latvian/mods/kubejs/fabric/BuiltinKubeJSFabricPlugin.java

```diff
@@ -1,10 +1,18 @@
 package dev.latvian.mods.kubejs.fabric;
 
 import dev.latvian.mods.kubejs.BuiltinKubeJSPlugin;
+import dev.latvian.mods.kubejs.item.custom.MultitoolItemJS;
+import dev.latvian.mods.kubejs.registry.RegistryInfo;
 import dev.latvian.mods.kubejs.script.ScriptType;
 import dev.latvian.mods.kubejs.util.ClassFilter;
 
 public class BuiltinKubeJSFabricPlugin extends BuiltinKubeJSPlugin {
+	@Override
+	public void init() {
+		super.init();
+
+		RegistryInfo.ITEM.addType("multitool", MultitoolItemJS.Builder.class, MultitoolItemJS.Builder::new);
```

Comment:
Why only Fabric?

Reply:
i also have [a class for forge](https://github.com/KubeJS-Mods/KubeJS/pull/681/commits/3f88795d68c501b3aa29e1509133c67c1902d9c4#diff-e5e3ace2152650752dcbf461f578a3cb672d62571d477b479a5bf7a03568174a), since it needs the toolaction things

Reply:
ah, I hadn't seen that